app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+from app.players.elves.elf_ranger import ElfRanger
+from app.players.elves.druid import Druid
+
+
+def calculate_team_total_rating(
+        players: list[
+            DwarfWarrior,
+            DwarfBlacksmith,
+            ElfRanger,
+            Druid
+        ]
+) -> int:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(
+        elves: list[
+            ElfRanger,
+            Druid
+        ]
```

Comment:
This will work for all Elfs so you can use `list[Elf]` instead of putting all classes.